README.md

```diff
@@ -60,10 +63,17 @@ _Note_: Feel free to use the `nightly` tag to test the latest version, i.e. `git
 ```bash
 tar -xzf openvscode-server-v${OPENVSCODE_SERVER_VERSION}.tar.gz
 cd openvscode-server-v${OPENVSCODE_SERVER_VERSION}
-./server.sh
+./bin/openvscode-server # you can add arguments here, use --help to list all of the possible options
```

Comment:
Here add clarification that by default the server listens to localhost only, so what users want usually is use `--host 0.0.0.0`